strr-platform-web/nuxt.config.ts

```diff
@@ -84,7 +84,9 @@ export default defineNuxtConfig({
       paymentPortalUrl: process.env.NUXT_PAYMENT_PORTAL_URL,
       baseUrl: process.env.NUXT_BASE_URL,
       environment: process.env.NUXT_ENVIRONMENT_HEADER || '',
-      version: `STRR PM UI v${process.env.npm_package_version}`
+      version: `STRR Platform UI v${process.env.npm_package_version}`,
+      ldClientId: process.env.NUXT_LD_CLIENT_ID || '',
+      appName: process.env.npm_package_name
```

Comment:
all this does is allow us to make different variations in ldarkly messages, etc. based on the application (for projects like strr where there are several different UIs using the same ldarkly project this can be useful)

Reply:
Not that it really matters but these config properties are set in the layer

Reply:
Oh yeah that's right -- was encountering an issue with my build not refreshing the .env and was thinking it was due to this. I will remove these

Reply:
updated